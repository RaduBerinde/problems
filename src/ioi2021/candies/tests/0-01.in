lrts0z0ktpqc670i0etyqgyb45dt1dysq5ap2mzg
3
10 15 13
2
0 2 20
0 1 -11
