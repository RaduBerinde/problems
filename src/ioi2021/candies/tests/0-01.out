4lpv73iykswp9e3nppi3jw2qggm5dfz29yy9nzpq
OK
0 4 13
